### PR TITLE
Use -[NSSet getObjects:] instead of -[NSSet getObjects:count:] to maintain compatibility with existing code passing NSArrays into this function

### PR DESCRIPTION
**Pull request by Catfish-Man**
Fixes rdar://57801853

### CODE REVIEW

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test

**Review by lorentey — APPROVED** [review]
😞

File: stdlib/public/core/ShadowProtocols.swift [hunk 1]
```diff
@@ -175,6 +175,10 @@ internal protocol _NSSet: _NSSetCore {
     _ buffer: UnsafeMutablePointer<AnyObject>,
```
[the remark this answers is not in the document]
Comment (Catfish-Man) [reply stdlib/public/core/ShadowProtocols.swift]: Possibly. It's not hurting anything though (NSSet does implement that), and I figure removing stuff is riskier than adding it.

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test

**Pull request merged**